posts/2021-01-06-ocp-log-forwarding.adoc

```diff
@@ -13,7 +13,9 @@ additional_authors:
   image: https://avatars0.githubusercontent.com/Channyboy
 ---
 = Forwarding Open Liberty logs in OpenShift to Splunk by using the Log Forwarding API
-Halim Lee <https://github.com/leemelim>
+Halim Lee <https://github.com/melissa-lee>
+
+**Important**: To use the dashboards that are provided in the following guide, you must configure the cluster logging Fluentd collectors to merge nested JSON data to the parent JSON payload. Although this configuration is required to use the Liberty dashboards, it can cause errors in the default OpenShift Elasticsearch instance. Elasticsearch dynamic mapping indexes a field and data type on a first-come basis. If multiple deployments generate JSON logs with the same fields, but are of different data types, merging the JSON data can potentially lead to data loss in the default Elasticsearch instance. Review your OpenShift configuration to determine whether the potential impact on the default Elasticsearch instance poses a risk to your environment. For more information about configuring Fluentd to merge JSON data, see <<integrating-open-liberty-json-logs,Integrating Open Liberty JSON logs>>.
```

Comment:
For this instance, should it be "this blog post", rather than "this guide"?

Reply:
Would _article_ apply to both an guide/documentation and a blog?🤔

Reply:
I think "article" works. Would it also be true to say

"To use the Liberty dashboards..."

I think for the docs, IBM style guidance advises against being self-referential ("this page", "this document"), though that isn't applied as stringently to guides and blog posts.

Reply:
Ah then "To use the Open Liberty dashboards that are provided, you must configure..."
I suppose for the OL blog we would should specify `Open Liberty` and can just use `Liberty` for the cp4a guide. And if the style guide (didn't know there was one!) says not to we can omit it?

Reply:
aren't the dashboards- the same for both Open Liberty and Liberty/CP4a ? If I remember correctly, for the kibana dashboards at least, there was just one GH repo called "Liberty Dashboards". If that's the case here, I'd recommend saying "Liberty" in OL doc as well, so user knows they're in the right place if they click through.
 
 They no self-referential guidance comes from Acrolinx- if you run the Acrolinx editor tool on a doc it flags anything like "this document", "this guide" etc.